bom/application/pom.xml

```diff
@@ -25,7 +25,7 @@
         <opentracing-tracerresolver.version>0.1.8</opentracing-tracerresolver.version>
         <opentracing-concurrent.version>0.2.0</opentracing-concurrent.version>
         <opentracing-jdbc.version>0.0.12</opentracing-jdbc.version>
-        <opentracing-kafka.version>0.1.13</opentracing-kafka.version>
+        <opentracing-kafka.version>0.1.15</opentracing-kafka.version>
```

Comment:
Should we add it to dependabot?

Reply:
Not sure, it contains a dependency to opentracing-api and we manage our own version (0.31 for the moment, latest is 0.33 I think), so we need to test it before upgrading it (I know that 0.1.15 works as I use it everyday).

Reply:
OK, makes perfect sense to keep it out of it then!